dimagi/pages/templates/sections/sectors/disease-treatment/part3.html

```diff
@@ -0,0 +1,37 @@
+{% load dimagidotcom %}
+{% load i18n %}
+{% block meta %}
+  <meta property="wistia:videoId" content="jzm3ggrinr" />
+  <meta property="wistia:videoTrigger" content="demo-video" />
+{% endblock %}
+
+<article class="epic part3">
+  <section class="epic-cover"
+    data-lazybg="true">
+    <div class="commcare-text pad-top-base pad-top-medium-at-medium pad-top-medium-at-large pad-top-medium-at-extra-large pad-top-base-at-small pad-bottom-medium pad-left-large-at-small">
+      <div class="content content-width-medium">
+        <div class=" content-width-small">
+          <h3 class="white mb-10">
+            {% blocktrans %}
+              See CommCare in Action
+            {% endblocktrans %}
+          </h3>
+          <p class="white">
+            {% blocktrans %}
+              Watch as our team takes you through the design of a basic
+              app in CommCare
+            {% endblocktrans %}
+          </p>
+          <p class="cta">
+            {% trans 'Watch Now' as video_text %}
+            {% click_hero 'video' as video_click_class %}
+            {% include "partials/wistia_popover.html" with wistia_id='jzm3ggrinr' show_play=True text=video_text click_class=video_click_class %}
+          </p>
+        </div>
+      </div> {# .content #}
+      <div class="phone-image off-grid white-lazy-bg"
```

Comment:
see comment I made about using the phone image here https://github.com/dimagi/dimagi.com/pull/92